core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/BasicAuthUIAuthenticator.java

```diff
@@ -161,10 +161,9 @@ public void unauthenticate(Object object) throws Exception {
         try {
             getAuthenticationAdminCient(((HttpServletRequest) object)).logout();
         } catch (AxisFault axisFault) {
-            String msg = "Configuration context is null.";
-            log.error(msg);
-            axisFault.printStackTrace();
-            throw new AxisFault(axisFault.getMessage());
+            String msg = "Failed to unauthenticated ";
```

Comment:
This message can be bit more expressive perhaps, it doesn't make sense at the moment.